themes/default/layouts/partials/footer.html

```diff
@@ -27,6 +27,7 @@
 
             <ul>
                 <li><a data-track="footer-about-us" href="{{ relref . "/about" }}" class="link">About Us</a></li>
+                <li><a data-track="footer-contact-us" href="{{ relref . "/request-a-demo" }}" class="link">Request a demo</a></li>
```

Comment:
I would make it "Request a Demo" given everything else is using some form of title case